runtime/menu.vim

```diff
@@ -80,6 +80,9 @@ if has("gui_macvim")
   an <silent> 9998.350 Window.Select\ Previous\ Tab :tabprevious<CR>
   an 9998.360 Window.-SEP2-			    <Nop>
   an <silent> 9998.370 Window.Bring\ All\ To\ Front <Nop>
+  an <silent> 9998.380 Window.Stay\ On\ Bottom <Nop>
```

Comment:
yeah, using same naming as Apple app sounds reasonable > "Stay in Front" and "Stay in Back"